src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -107,6 +109,10 @@ ccjRequestedForDefendantClaimDetails %}
       {% elif state.status === ClaimStatus.OFFER_SETTLEMENT_REACHED
               or state.status === ClaimStatus.ELIGIBLE_FOR_CCJ_AFTER_BREACHED_SETTLEMENT %}
         {{ offerSettlementReachedForClaimant(claim) }}
+      {% elif state.status === ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_AS_COMPANY_OR_ORGANISATION_RESPONSE %}
+        {{ claimantRejectedDefendantAsCompanyOrOrganisationResponseDetailedStatus(claim) }}
+      {% elif state.status === ClaimStatus.ADMISSION_SETTLEMENT_AGREEMENT_REACHED %}
+        Show this status
```

Comment:
this doesn't look like it should be here. perhaps you were using it for testing?